piikii-application/src/main/kotlin/com/piikii/application/domain/course/Distance.kt

```diff
@@ -0,0 +1,10 @@
+package com.piikii.application.domain.course
+
+data class Distance(
+    val totalDistanceMeter: Int?,
+    val totalTimeMinute: Int?,
+) {
+    companion object {
+        fun empty() = Distance(null, null)
```

Comment:
이거 함수로 안하고 상수로 선언해도 될 것 같은데요!?
```suggestion
        val EMPTY = Distance(null, null)
```

companion object method로 생성하면 컴파일됬을 때 클래스 단위로 돌아가서 불필요한 클래스가 생겨나지 않을까 싶습니다 (찾아보시길 추천).